scraper.js

```
@@ -0,0 +1,695 @@
+import puppeteer from 'puppeteer';
+import { getProductModel } from './models/Product.js';
+import './db.js';
```

Add error handling for database connection.

Importing db.js for side effects without error handling could lead to silent failures. Consider importing and explicitly handling the connection.